### PR TITLE
update hugo to v0.117.0

### PR DESCRIPTION
**Pull request by Arhell**
# What issue does it fix
Closes https://github.com/getporter/porter/issues/2859

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   command = "go run mage.go -v Docs"
 
 [build.environment]
-  HUGO_VERSION = "0.115.4"
+  HUGO_VERSION = "0.116.1"
```
Comment (troy0820) [inline netlify.toml:15]: This looks good and thank you for fixing this issue. Can you update the Hugo version in the CONTRIBUTING.md as well.

**Review by troy0820 — CHANGES_REQUESTED** [review]
Just the Hugo version in the contributing doc and any references to it within the code.

File: netlify.toml (hunk 1 above)
Comment (Arhell) [reply netlify.toml:15]: Done thanks

**Review by troy0820 — COMMENTED** [review]
LGTM

**Review by troy0820 — APPROVED** [review]

**Review by troy0820 — CHANGES_REQUESTED** [review]
I was looking at this again to approve but forgot one line that needs to be updated 
https://github.com/getporter/porter/blob/main/netlify.toml#L15

This will deploy the website with the latest hugo.  @Arhell

**Review by Arhell — COMMENTED** [review]
Done

**Review by troy0820 — APPROVED** [review]

**Pull request merged**